<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NzbGetSettingsDialog</class>
 <widget class="QDialog" name="NzbGetSettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>360</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>NZBGet-instellingen</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QCheckBox" name="enableCheckBox">
     <property name="text">
      <string>NZBGet integratie inschakelen</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="settingsGroup">
     <property name="title">
      <string>Serverinstellingen</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="hostLabel">
        <property name="text">
         <string>Host</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="hostEdit"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="portLabel">
        <property name="text">
         <string>Poort</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="portSpin">
        <property name="maximum">
         <number>65535</number>
        </property>
        <property name="value">
         <number>6789</number>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QCheckBox" name="sslCheckBox">
        <property name="text">
         <string>Gebruik SSL (https)</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="userLabel">
        <property name="text">
         <string>Gebruikersnaam</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="usernameEdit"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="passLabel">
        <property name="text">
         <string>Wachtwoord</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="passwordEdit">
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="categoryLabel">
        <property name="text">
         <string>Categorie</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLineEdit" name="categoryEdit"/>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="priorityLabel">
        <property name="text">
         <string>Prioriteit</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QSpinBox" name="prioritySpin">
        <property name="minimum">
         <number>-1000</number>
        </property>
        <property name="maximum">
         <number>1000</number>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
      <widget class="QCheckBox" name="pausedCheckBox">
        <property name="text">
         <string>Toevoegen als pauze</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="testButton">
       <property name="text">
        <string>Test verbinding</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>NzbGetSettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>315</y>
    </hint>
    <hint type="destinationlabel">
     <x>273</x>
     <y>339</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>NzbGetSettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>135</x>
     <y>315</y>
    </hint>
    <hint type="destinationlabel">
     <x>111</x>
     <y>339</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
